tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -341,8 +353,10 @@
         - run-os
         - install-certs
         - ceph-client
```

Comment:
seems we would also  not need ceph-client on networkers, any reason to keep it outside of if block below?